renovate.json5

```
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "branchPrefix": "renovate/",
+  "extends": [
+    "config:base",
+    "github>coveo/renovate//presets/coveo-defaults.json5",
+    "schedule:earlyMondays",
+    "group:allNonMajor"
+  ],
+  "prConcurrentLimit": 3,
+  "prHeader": "DT-5590 <- Put your own jira issue here by setting the `prHeader` value in your renovate config",
```

I think we need to change this, should we use the DPF Renovate JIRA for this?
```suggestion
  "prHeader": "UA-5791",
```

---

Yep, that's what all our other repos use